samples/MortgageApplication/application-conf/file.properties

```diff
@@ -7,6 +7,7 @@ dbb.scriptMapping = BMS.groovy :: **/*.bms
 dbb.scriptMapping = Cobol.groovy :: **/*.cbl
 dbb.scriptMapping = LinkEdit.groovy :: **/*.lnk
 dbb.scriptMapping = PLI.groovy :: **/*.pli
+dbb.scriptMapping = CRB.groovy :: **/*.yml, **/*.yaml
```

Comment:
Please remove the scriptMapping from the MortgageApplication sample.

Reply:
Fixed

Reply:
Fixed